Ghidra/Features/Base/src/main/java/ghidra/app/util/demangler/Demangled.java

```diff
@@ -20,9 +20,9 @@
  * 
  * <p>This class and its children have many overlapping concepts that we wish to refine at a 
  * future date.  Below is a listing of known uses:
- * <TABLE>
+ * <TABLE><CAPTION></CAPTION>
```

Comment:
Javadoc generates a warning about the empty caption. However it generates an error if there is no summary or caption. It also generates an error if summary is used as it is not available in html5 :man_shrugging: